<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>C project</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_8">
    <item row="2" column="0" colspan="3">
     <widget class="QPushButton" name="pushButtonExit">
      <property name="text">
       <string>E&amp;xit</string>
      </property>
      <property name="iconSize">
       <size>
        <width>15</width>
        <height>15</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Project</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Project</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_2">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="text">
                <string>Location</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1" colspan="2">
              <widget class="QLineEdit" name="lineEditProjectLocation">
               <property name="readOnly">
                <bool>true</bool>
               </property>
               <property name="placeholderText">
                <string/>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QToolButton" name="toolButtonLocation">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="1" column="0">
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="0">
              <widget class="QPushButton" name="pushButtonNew">
               <property name="text">
                <string>&amp;New</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QPushButton" name="pushButtonLoad">
               <property name="text">
                <string>&amp;Load</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QPushButton" name="pushButtonUpdate">
               <property name="text">
                <string>&amp;Update</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QGroupBox" name="groupBox_4">
          <property name="title">
           <string>Target Memory</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditMemorySize">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>16384</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Memory Size</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditDataSegSize">
             <property name="inputMask">
              <string/>
             </property>
             <property name="text">
              <string>2048</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Data Segment Size</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>Optimization Level</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_6">
           <item row="0" column="0">
            <widget class="QComboBox" name="comboBoxOptimization">
             <property name="layoutDirection">
              <enum>Qt::LeftToRight</enum>
             </property>
             <item>
              <property name="text">
               <string>Default (-O0)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Minimal (-O1)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>More (-O2)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Standard (-O3)</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>For Size (-Os)</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_3">
          <property name="title">
           <string>Compiler</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <layout class="QGridLayout" name="gridLayout_4">
           <item row="0" column="0">
            <layout class="QGridLayout" name="gridLayout_2">
             <item row="0" column="0">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Prefix</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QLineEdit" name="lineEditCompilerPrefix">
               <property name="text">
                <string>mips-sde-elf-</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Location</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="lineEditCompilerPath">
               <property name="text">
                <string>/opt/mgc/embedded/codebench</string>
               </property>
               <property name="placeholderText">
                <string>&lt;if not in path&gt;</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
        <item row="4" column="0">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Bootloader</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_13">
        <item row="0" column="0">
         <widget class="QGroupBox" name="groupBox_7">
          <property name="title">
           <string>Serial Port</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_12">
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButtonOpenPort">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="text">
              <string>&amp;Open</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QComboBox" name="comboBoxSerialPort"/>
           </item>
           <item row="0" column="1">
            <widget class="QPushButton" name="pushButtonReload">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="text">
              <string>&amp;Reload</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QGroupBox" name="groupBox_6">
          <property name="title">
           <string>Bin File</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_11">
           <item row="0" column="1">
            <widget class="QToolButton" name="toolButtonSeletBinFile">
             <property name="text">
              <string>...</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLineEdit" name="lineEditBinFile">
             <property name="readOnly">
              <bool>true</bool>
             </property>
             <property name="placeholderText">
              <string>Select BIN File</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QPushButton" name="pushButtonBoot">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="focusPolicy">
              <enum>Qt::TabFocus</enum>
             </property>
             <property name="text">
              <string>&amp;Boot</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="2" column="0">
         <widget class="QGroupBox" name="groupBox_5">
          <property name="title">
           <string>Console</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_10">
           <item row="0" column="0">
            <widget class="QTextEdit" name="textEdit">
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="pushButtonClear">
             <property name="text">
              <string>&amp;Clear</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="labelIcon">
      <property name="maximumSize">
       <size>
        <width>220</width>
        <height>73</height>
       </size>
      </property>
      <property name="text">
       <string>Insert Logo</string>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEditProjectLocation</tabstop>
  <tabstop>pushButtonNew</tabstop>
  <tabstop>pushButtonLoad</tabstop>
  <tabstop>pushButtonUpdate</tabstop>
  <tabstop>lineEditCompilerPrefix</tabstop>
  <tabstop>lineEditCompilerPath</tabstop>
  <tabstop>comboBoxOptimization</tabstop>
  <tabstop>lineEditMemorySize</tabstop>
  <tabstop>lineEditDataSegSize</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
